src/components/App/index.tsx

```diff
@@ -19,11 +19,13 @@ const globalStyles = (th: Theme) =>
       padding: 0,
       backgroundColor: th.colors.background1,
       color: th.colors.text1,
-      ...th.typography.variants.body1,
-    },
+      ...th.typography.variants.body1
+    }
   })
 
-const store = makeStore()
+const urlParams = new URLSearchParams(window.location.search)
```

Comment:
I think we are using `react-router-dom` in the project, can it handle search params here?

Reply:
Good point! Removed it from Redux and wrapped in a `useFeatureFlags` hook, that calls `useLocation` from ReactRouter. Looks like there is a MemoryRouter for testing purposes too, but I did not try it out, since we don't have tests anyway 🙂 
It was nice to avoid the empty reducer hack too btw 👍